src/commands.ts

```diff
@@ -594,6 +594,14 @@ export function registerCommands(context: vscode.ExtensionContext, reposManager:
 		});
 	}));
 
+	context.subscriptions.push(vscode.commands.registerCommand('pr.setFileListLayoutAsTree', _ => {
+		vscode.workspace.getConfiguration('githubPullRequests').update('fileListLayout', 'tree');
```

Comment:
Sorry, one last thing that I missed! There is an optional argument to `update` that we should set to `true` at the end of this. This controls whether the setting is updated for the workspace, or for the user's overall settings. If the update it done just for the workspace, it creates a file in that folder. We just want to update the user's settings

Reply:
No worries! Thank you for explaining the reasoning behind it. I've made the changes pushed them